ajax/libs/emojionearea/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "emojionearea", 
+  "description": "EmojiOne Area is a small jQuery plugin that allows you to transform any html element into simple WYSIWYG editor with ability to use Emojione icons. The end result is a secure text/plain in which the image icons will be replaced with their Unicode analogues.",
+  "filename": "emojionearea.min.js",
+  "keywords": [
+	"emoji", 
+	"emojione", 
+	"picker", 
+	"emojionearea"
+  ],
+  "author": "Andrey Izman <izmanw@gmail.com>",
+  "homepage": "http://mervick.github.io/emojionearea/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mervick/emojionearea.git"
+  },
+  "autoupdate": {
+    "source": "git",
```

Comment:
Let's use npm auto-update
Because there are 43 releases versions from npm, and 32 releases versions from GitHub.
As usual, we prefer to use more releases versions as the auto-update option.
Thank you.

Reply:
@sufuf3 Please check again. npm source should be https://www.npmjs.com/package/emojionearea instead of https://www.npmjs.com/package/emojione. So we can just use git as the source.

Reply:
Got it.
It's fine here.